app/src/main/java/com/napzak/market/presentation/explore/ExploreRoute.kt

```diff
@@ -0,0 +1,232 @@
+package com.napzak.market.presentation.explore
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.napzak.market.core.common.state.UiState
+import com.napzak.market.core.designsystem.component.textField.SearchBox
+import com.napzak.market.domain.explore.model.ProductItem
+import com.napzak.market.presentation.explore.state.ExploreUiState
+import com.napzak.market.R
+import com.napzak.market.core.common.extension.noRippleClickable
+import com.napzak.market.core.designsystem.theme.NapzakMarketTheme
+import com.napzak.market.presentation.explore.component.ExploreFilterGroup
+import com.napzak.market.presentation.explore.component.ProductListSection
+import com.napzak.market.presentation.explore.component.TradeTypeTab
+import com.napzak.market.presentation.explore.type.SortType
+import com.napzak.market.presentation.explore.type.TradeType
+import com.napzak.market.presentation.explore.type.getSortLabel
+
+@Composable
+fun ExploreRoute(
+    onSearchBoxClick: () -> Unit,
+    navigateToProductDetail: () -> Unit,
+    modifier: Modifier = Modifier,
+    viewModel: ExploreViewModel = viewModel()
+) {
+    val uiState by viewModel.uiState.collectAsStateWithLifecycle()
+
+    LaunchedEffect(true) {
+        viewModel.getExploreProductInformation()
+    }
+
+    ExploreScreen(
+        modifier = modifier,
+        uiState = uiState,
+        onSearchBoxClick = onSearchBoxClick,
+        onTradeTypeClick = { viewModel.changeTradeType(it) },
```

Comment:
여기처럼 onTradeTypeClick의 인자와 viewModel.changeTradeType의 인자 개수, 타입, 순서가 같은 경우 람다를 사용하지 않고 `::`연산자로 객체의 함수를 불러올 수 있어요!
```suggestion
        onTradeTypeClick = viewModel::changeTradeType,
```